troposphere/networkfirewall.py

```diff
@@ -114,17 +114,28 @@ class FirewallPolicy(AWSObject):
     }
 
 
-class LogDestinationConfig(AWSProperty):
+class LogDestinationConfigs(AWSProperty):
```

Comment:
According to the docs, this should remain singular.

Reply:
Ahh, you are correct.  Updated!